blocks/footer/render.php

```diff
@@ -26,7 +26,7 @@
 				<span class="wp-block-wp-newsletter-builder-footer__social-links__item">
 					<a class="wp-block-wp-newsletter-builder-footer__social-links__link facebook-icon" href="<?php echo esc_url( $nb_facebook_url ); ?>">
 						<img
-							src="<?php echo esc_url( home_url() . '/wp-content/plugins/wp-newsletter-builder/images/facebook.png' ); ?>"
+							src="<?php echo esc_url( site_url() . '/wp-content/plugins/wp-newsletter-builder/images/facebook.png' ); ?>"
```

Comment:
there's probably a safer way to handle these... maybe https://developer.wordpress.org/reference/functions/plugin_dir_url/ ?

Reply:
@mogmarsh I ended up using `plugins_url` because `plugin_dir_url` got weird because compiled block files sit in the build directory. Let me know what you think. commit a8eb2d5

Reply:
@efuller looks good